src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,144 @@
+//SCSS variables
+$gray_color: #777;
+$lightgray_color: #D8D8D8;
+$sky_color: #00BFFF;
+$font_nova: 'Nova Flat';
+$gray_border: 0.19rem solid $gray_color;
+$lightgray_border: 0.0625rem solid $lightgray_color;
```

Comment:
you can use 1px and 2px for borders when needed, rem is not convenient for that, for everything else use rem